assets/src/blocks/widgets/event-attendees/edit.js

```diff
@@ -0,0 +1,263 @@
+
+/**
+ * Internal dependencies
+ */
+import { EventAttendeesList } from './list';
+
+class EventAttendeesEditor extends Component {
+	static propTypes = {
+		attendees: PropTypes.array,
+		isLoading: PropTypes.bool,
+		attributes: PropTypes.shape( {
+			eventId: PropTypes.number,
+			datetimeId: PropTypes.number,
+			ticketId: PropTypes.number,
+			status: PropTypes.string,
+			showGravatar: PropTypes.bool,
+			displayOnArchives: PropTypes.bool,
+			limit: PropTypes.number,
+		} ),
+	};
+
+	static defaultProps = {
+		attendees: [],
+		isLoading: true,
+		attributes: {
+			eventId: 0,
+			datetimeId: 0,
+			ticketId: 0,
+			status: 'RAP',
+			showGravatar: true,
+			displayOnArchives: false,
+			limit: -1
+		},
+	};
+
+	setEventId = ( eventId ) => {
+		const value = eventId !== null && eventId.value ?
+			parseInt( eventId.value, 10 ) :
+			0;
+		this.props.setAttributes(
+			{
+				eventId: value,
+				datetimeId: 0,
+				ticketId: 0,
+			}
+		);
+	};
+
+	setDatetimeId = ( datetimeId ) => {
+		const value = datetimeId !== null && datetimeId.value ?
+			parseInt( datetimeId.value, 10 ) :
+			0;
+		this.props.setAttributes(
+			{
+				datetimeId: value,
+				ticketId: 0,
+			}
+		);
+	};
+
+	setTicketId = ( ticketId ) => {
+		const value = ticketId !== null && ticketId.value ?
+			parseInt( ticketId.value, 10 ) :
+			0;
+		this.props.setAttributes( { ticketId: value } );
+	};
+
+	setStatus = ( status ) => {
+		const value = status !== null && status.value ? status.value : 'ANY';
+		console.log( 'status: ' + value );
+		this.props.setAttributes( { status: value } );
+		console.log( 'this.props.attributes.status: ' + this.props.attributes.status );
+	};
+
+	setLimit = ( limit ) => {
+		this.props.setAttributes( {
+			limit: parseInt( limit, 10 )
+		} );
+	};
+
+	toggleShowGravatar = ( showGravatar ) => {
+		this.props.setAttributes( { showGravatar: showGravatar } );
+	};
+
+	toggleDisplayOnArchives = ( displayOnArchives ) => {
+		this.props.setAttributes( { displayOnArchives: displayOnArchives } );
+	};
+
+	getPlaceHolderContent() {
+		if ( ! this.props.isLoading && this.props.attributes.eventId ) {
+			return (
+				<p> { __( 'No attendees... yet!', 'event_espresso' ) } </p>
+			);
+		}
+		if ( ! this.props.isLoading && ! this.props.attributes.eventId ) {
+			return (
+				<p>
+					{ __(
+						'Please select an event to display attendees for.',
+						'event_espresso'
+					) }
+				</p>
+			);
+		}
+		return <Spinner />;
+	}
+
+	getNoAttendeesContent() {
+		return (
+			<Fragment>
+				<Placeholder
+					icon="groups"
+					label={ __( 'Event Attendees Block', 'event_espresso' ) }
+				>
+					{ this.getPlaceHolderContent() }
+				</Placeholder>
+			</Fragment>
+		);
+	}
+
+	getAttendeesDisplay() {
+		return (
+			<EventAttendeesList attendees={ this.props.attendees }/>
+		);
+	}
+
+	render() {
+		const { attributes } = this.props;
+		let { attendees } = this.props;
+		attendees = uniqBy( attendees, 'ATT_ID' );
```

Comment:
ya I was getting duplicates in the actual displayed list

Reply:
I think we may want to try and figure out why you are getting duplicates.  Is it possible that this is leftover from when you querying against the registration endpoint instead of the attendee endpoint?

Reply:
ya very likely.. that would make sense